### ISSUE TITLE
Support for Unity Free

### ISSUE THREAD

**Issue by darkcrawler01** [Busta117/SQLiteUnityKit#5]
I didnt see this in the Readme. Does this library require Unity Pro to connect ?

**Comment by lyonmen** [comment]
Im using this library for one week now, dosent require Unity Pro, my version is the Free Unity.

**Comment by darkcrawler01** [comment]
where you also able to build for ios/android successfully ?

**Comment by lyonmen** [comment]
yes for Android, i didnt try o ios yet. My only problem is to create the databasename.db to add to project, i have on created and i dont remember how i created. im workin on it right now

**Comment by darkcrawler01** [comment]
Weird .. It says that it uses system.net.sockets which requires pro
license.

**Comment by lyonmen** [comment]
here the files of my project with the files provide by this library + my script and the database created for the test. http://lyonmen.com.br/sqlite3teste.rar
Create a new Project just to be sure. If you got any problems let me know.
Dont forget to attach the Sqlite3Teste.cs to your camera.

How use test the App:
Button 1 is to show the result from the button 3 and clear the button 1 when its pressed.
Button 2 is for insert one record on the Database.
Button 3 is for Read the Record from Database and send the data to button 1
Button 4 is for Delete the Record from the database and clear the Button 1
Button 5 TRUE = The database will be deleted and created again when the app will start on next time.
Button 5 False = The database will NOT be deleted when the app will start on next time.

**Comment by darkcrawler01** [comment]
thanks man ! appreciate the effort :+1:

**Comment by darkcrawler01** [comment]
Sorry I closed the issue w/o testing. Still the same error 

License error. This plugin is only supported in Unity Pro!

SqliteDatabase:ExecuteNonQuery(String) (at Assets/sqlite3teste/SqliteDatabase.cs:192)
SqliteDatabase:ExecuteNonQuery(String) (at Assets/sqlite3teste/SqliteDatabase.cs:191)
Sqlite3Teste:CreateDatabase() (at Assets/sqlite3teste/Sqlite3Teste.cs:89)
Sqlite3Teste:Start() (at Assets/sqlite3teste/Sqlite3Teste.cs:17)

**Comment by lyonmen** [comment]
Thats Strange, but i remember i got that same error some time ago. but i dont remember if was in this plugin or another that i got this error.
I will be honest, im a quite a noob on developing for Android and ios.

I just did the same test again, i created a new Project from scratch. Copy all content of my .rar inside the Asset folder. Add the Sqlite3Teste.cs to my Camera. Put my developer key on the publishing settings. 
Set the Company name, Product Name and Bundle Identifier, and hit to build.
No errors.

just for record: You Dont need the Sqlite3.dll

My Version of Unity is 4.5.4

as i coment, im quite a noob and i dont know if the Java versions have some Important on it, so my Java is:
Jdk: 1.8.0_25  32bits
Jre: 1.8.0_25  32bits
I also have the Eclipse ADT with the Android Sdk for Windows 32bits from this site: https://developer.android.com/sdk/index.html?hl=i#download
 I also add on Ambient Variables on Windows the variable PATH with the folder to my Java (i install the java on diferente folder)
My Windows is 8.1 pro 64bits

This is all my configuration

be sure to start a new Project, i read somewhere that some times the Unity may left some dump on the folders who can made erros

**Comment by darkcrawler01** [comment]
http://docs.unity3d.com/Manual/Plugins.html

The DLLImport seems to be a Pro only feature.

**Comment by lyonmen** [comment]
As i say one time, i remember to got that same error, but now it works. 
I try to find some related on internet and there are people who says "you can" and "you cant" use DLLImport. check this link for more info related. 
http://forum.unity3d.com/threads/can-i-use-dlls-in-unity-free.56013/
I cant remember if did something special to pass this error.
I just checked the Editor folder on my Unity folder and i found one sqlite3.dll. i dont remember if this file is from unity itself or if i copy that dll to the folder.
Check for this file, and good luck.
I really want to help you more, but i already write everithing what i did. And i did this 2 times, on my home pc and work pc.

**Comment by darkcrawler01** [comment]
ok ! i tried the library in a different laptop and it worked ..  :weary:

**Comment by lyonmen** [comment]
Now the question is: what is diferente in your first test from that who worked?
Strange things happens! 

For your first test i will recomend to unistall everithing, Java, Unity, SDKs, and so on, and install all again. Setup a new project on diferent folder than usual and try again. 

As you see, it work on Unity free, but no one on Internet have the Real answer to the question.
If the first test was in your Default computer to work on unity, i recomend you to keep trying. 
At least now you know for sure this library really works on Unity free.
